packages/dropbox.rb

```diff
@@ -1,59 +1,30 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Dropbox < Package
-  description 'Dropbox simplifies the way you create, share and collaborate.  Bring your photos, docs, and videos anywhere and keep your files safe.'
+class Dropbox < Autotools
+  description 'Dropbox Integration for Nautilus'
```

Comment:
Why the description change?  Dropbox is not just for Nautilus.

Reply:
I think we should leave the description unchanged but add integration for Nautilus.